### DISCUSSION TITLE
Anydesk MSI and EXE?

### DISCUSSION THREAD

**Discussion by Trenly** [microsoft/winget-pkgs#33494] [Q&A] [answered]
Does anyone know if there is a reason AnyDeskSoftwareGmbH.AnyDesk and AnyDeskSoftwareGmbH.AnyDeskMSI are separate? Could they be merged to have both installers in the same file, or should they?

**Comment by KaranKad** [comment, marked as answer]
I think thats because exe and msi version have different version in ARP, exe has `ad 6.3.3` and msi had `6.3.3`, `winget upgrade` wont work properly if both of them are in same file, same issue with 7zip. I think manifest v1.1.0 might solve this issue.

**Reply by Trenly** [reply]
Ahhh, makes sense. I hadnt noticed that, Thank you
